package.json

```diff
@@ -36,7 +36,8 @@
     "openapi-generator": "openapi-generator generate -g typescript-fetch --additional-properties=typescriptThreePlus=true",
     "schemas": "node ./scripts/schemas.js",
     "postinstall": "node ./scripts/postinstall.js",
-    "gitbook": "node ./scripts/gitbook.js"
+    "gitbook": "node ./scripts/gitbook.js",
+    "secret-service": "./scripts/create-secret.sh"
```

Comment:
We might want to rename these script tags, since we also have
`"env-secrets": "./scripts/secrets.sh",`.

That one fetches development secrets from the param store and loads them to env.

so `yarn (fetch/get/load)-secrets` and `yarn create-secret` is probably more descriptive.

Reply:
makes sense, fixing.